frontend/packages/noobaa-storage-plugin/src/components/data-resiliency-card/data-resiliency-card.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import { Icon } from 'patternfly-react';
+import { Progress, pluralize } from '@patternfly/react-core';
+
+import './data-resiliency-card.scss';
+
+import { DashboardCard } from '@console/internal/components/dashboard/dashboard-card/card';
+import { DashboardCardBody } from '@console/internal/components/dashboard/dashboard-card/card-body';
+import { DashboardCardHeader } from '@console/internal/components/dashboard/dashboard-card/card-header';
+import { DashboardCardTitle } from '@console/internal/components/dashboard/dashboard-card/card-title';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { DashboardCardHelp } from '@console/internal/components/dashboard/dashboard-card';
+
+import { DATA_RESILIENCE_QUERIES } from '../../queries';
+import { getPropsData } from '../../utils';
+
+const getFormattedEta = (eta) => {
```

Comment:
it can be in the utils file inside `data-resiliency-card` and later required can be moved to common utils. It's totally up to you.

Reply:
This function is just for data resiliency, I dont think it will be required as utility somewhere else.
@cloudbehl

Reply:
Right now not using it, but if later required we can move it at that point of time.